### PR TITLE
Feature/Adding initial data

### PR DESCRIPTION
**Pull request by Pjaijai**

### CODE REVIEW

File: supabase/seed.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  (14, 'eddaced1-4336-473f-9268-82a0bd51957d', 'charlottetown', '0f66dd8f-22d2-4977-9f5e-54149750d330', 'Charlottetown', '夏洛特敦'),
+  (15, 'db984e01-6d4a-4f1f-81ff-2589aed69044', 'regina', '0d38e506-4421-4283-bcb5-28e0bf0caf03', 'Regina', '雷吉納'),
+  (16, '2e5d4fcf-fc67-47d8-b8fb-c9e671fae44c', 'saskatoon', '0d38e506-4421-4283-bcb5-28e0bf0caf03', 'Saskatoon', '沙斯加通'),
+  (17, 'f8f68409-5789-44c3-8684-d7c6fa656623', 'St_Johns', '1cd459f0-0e0f-45fe-9c4b-ec7e0ce22340', 'St. Johns', '聖約翰'),
+  (18, 'b2528cc2-21e2-4c5b-af1f-628aabcae9b1', 'waterloo', '7546ebd4-6057-4d63-b1ae-40a8ee520c33', 'Waterloo', '滑鐵盧');
+
+-- Insert data into the "industry" table
+INSERT INTO industry (id, uuid, value, english_name, cantonese_name)
+VALUES
+  (3, '4b2af2dd-e72c-4618-a928-fbde3c427599', 'education', 'Education', '教育業'),
```
Comment (511234) [inline supabase/seed.sql:45]: should sort values according to id?

Comment (Pjaijai) [reply supabase/seed.sql:47]: updated. I let ChatGPT to it lol

File: supabase/seed.sql [hunk 2]
```diff
@@ -0,0 +1,69 @@
+-- Insert data into the "city" table
+-- Insert data into the "country" table
+INSERT INTO country (id, uuid, value, english_name, cantonese_name)
+VALUES
+  (1, 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'canada', 'Canada', '加拿大');
+
+-- Insert data into the "province" table
+INSERT INTO province (id, uuid, value, country_uuid, english_name, cantonese_name)
+VALUES
+  (1, '090d3e23-d1bb-4fff-b6c0-87fc4da6304c', 'alberta', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'Alberta', '亞伯達省'),
```
Comment (511234) [inline supabase/seed.sql:10]: also not sorted here

File: supabase/seed.sql [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  (5, '42a12df3-f785-44e7-9aca-208602b1544c', 'nova_scotia', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'Nova Scotia', '新斯科細亞'),
+  (6, '2a44e69f-3ae1-45ea-8dd3-572663b9e303', 'new_brunswick', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'New Brunswick', '新不倫瑞克省'),
+  (7, '9670157c-7370-4caa-86c6-4e73d0218796', 'manitoba', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'Manitoba', '馬尼托巴'),
+  (8, '0f66dd8f-22d2-4977-9f5e-54149750d330', 'prince_edward_island', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'Prince Edward Island', '愛德華王子島'),
+  (9, '0d38e506-4421-4283-bcb5-28e0bf0caf03', 'saskatchewan', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'Saskatchewan', '沙斯卡寸旺省'),
+  (10, '1cd459f0-0e0f-45fe-9c4b-ec7e0ce22340', 'newfoundland_and_labrador', 'c55a2a85-bd17-423d-b49f-548dc859ab67', 'Newfoundland and Labrador', '紐芬蘭及拉布拉多');
+
+INSERT INTO city (id, uuid, value, province_uuid, english_name, cantonese_name)
+VALUES
+  (1, '69a46dce-b0a1-4ba7-94f1-f6486ed48ca4', 'calgary', '090d3e23-d1bb-4fff-b6c0-87fc4da6304c', 'Calgary', '卡加里'),
```
Comment (511234) [inline supabase/seed.sql:23]: sort here

File: supabase/seed.sql [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  (9, '82a8511d-7f92-4e7b-a90e-d48a0bc5a445', 'halifax', '42a12df3-f785-44e7-9aca-208602b1544c', 'Halifax', '哈利法克斯'),
+  (10, '4de0e48e-fb8f-49ae-9966-85e12e60eeec', 'fredericton', '2a44e69f-3ae1-45ea-8dd3-572663b9e303', 'Fredericton', '弗雷德里克頓'),
+  (11, '6f02d216-2c64-4fa0-a8d2-37c9c43c7ad8', 'moncton', '2a44e69f-3ae1-45ea-8dd3-572663b9e303', 'Moncton', '蒙克頓'),
+  (12, 'b49f90a4-214c-4068-adb4-decc506dadbd', 'winnipeg', '9670157c-7370-4caa-86c6-4e73d0218796', 'Winnipeg', '溫尼伯'),
+  (13, '3f5534ce-ad2a-4b52-a07d-5efc2c86ed88', 'victoria', '62f381bd-c061-4c89-a9eb-7a902a69dfe9', 'Victoria', '維多利亞'),
+  (14, 'eddaced1-4336-473f-9268-82a0bd51957d', 'charlottetown', '0f66dd8f-22d2-4977-9f5e-54149750d330', 'Charlottetown', '夏洛特敦'),
+  (15, 'db984e01-6d4a-4f1f-81ff-2589aed69044', 'regina', '0d38e506-4421-4283-bcb5-28e0bf0caf03', 'Regina', '雷吉納'),
+  (16, '2e5d4fcf-fc67-47d8-b8fb-c9e671fae44c', 'saskatoon', '0d38e506-4421-4283-bcb5-28e0bf0caf03', 'Saskatoon', '沙斯加通'),
+  (17, 'f8f68409-5789-44c3-8684-d7c6fa656623', 'St_Johns', '1cd459f0-0e0f-45fe-9c4b-ec7e0ce22340', 'St. Johns', '聖約翰'),
```
[the remark this answers is not in the document]
Comment (Pjaijai) [reply supabase/seed.sql:39]: Updated. Also made changes on staging and prod. Good catch. Many thanks.

File: supabase/seed.sql (hunk 2 above)
Comment (Pjaijai) [reply supabase/seed.sql:10]: Updated. Thank you for pointing out.

File: supabase/seed.sql (hunk 3 above)
Comment (Pjaijai) [reply supabase/seed.sql:24]: Updated. Thank you for pointing out.

**Review by 511234 — DISMISSED** [review]

**Review by ariesscy — APPROVED** [review]

**Pull request merged**